### PR TITLE
UC-204 | Unified message on Special:Upload and Special:WikiaVideoAdd

### PR DESCRIPTION
**Pull request by kvas-damian**
Use the same mechanism for permission check on Special:Upload and Special:WikiaVideoAdd

### CODE REVIEW

File: languages/messages/MessagesQqq.php [hunk 1]
```diff
@@ -3772,4 +3772,6 @@
 
 {{Identical|Other}}',
 
+'uploadnologintext' => 'Information that user have to log in to upload files or add videos. Displayed for non logged in users that perform upload action. 1st parameter is LogIn page link.',
```
Comment (jsutterfield) [inline languages/messages/MessagesQqq.php]: Should probably be "Information that the user has to log in to upload files or add videos. Displayed for non-logged in users that perform upload action. 1st parameter is LogIn page link." It's slightly more grammatically correct.

File: extensions/wikia/VideoEmbedTool/WikiaVideoAdd_body.php [hunk 2]
```diff
@@ -12,23 +12,22 @@ function __construct () {
 		parent::__construct( "WikiaVideoAdd", "wikiavideoadd" );
 	}
 
+	protected function checkPermission() {
+		$permissionRequired = UploadBase::isAllowed( $this->getUser() );
+		if( $permissionRequired !== true ) {
+			throw new PermissionsError( $permissionRequired );
```
Comment (jsutterfield) [inline extensions/wikia/VideoEmbedTool/WikiaVideoAdd_body.php]: Didn't know this existed. That's cool.

**Comment by armonr** [comment]
Please add PR descriptions as per [Pull Request Guidelines](https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md#guidelines)

**Comment by armonr** [comment]
:+1:

**Pull request merged**